external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -86,6 +86,16 @@
     <PackageToInclude Include="$(XUnitRunnerPackageId)" />
   </ItemGroup>
 
+  <PropertyGroup Condition="'$(TargetGroup)' == 'uap'">
+    <UAPToolsPackageVersion>1.0.5</UAPToolsPackageVersion>
```

Comment:
It doesn't really, but we will never use it except on our uap vertical, so I thought that we could avoid downloading it unless we knew we needed it.